code/game/objects/items/toys.dm

```diff
@@ -798,6 +798,38 @@
 	desc = "WE GOT THE VALIDS AI CALL THE SHUTTLE"
 	icon_state = "comdom"
 
+/obj/item/toy/gasha/maniac
+	name = "toy maniac"
+	desc = "NOW WITH REAL KUNG-FU SEIZURE ACTION!"
+	icon_state = "maniac"

```

Comment:
This was added before he came back, it's not meant to be a metaclub thing you dork